src/postcss-values-parser.d.ts

```diff
@@ -1,26 +1,121 @@
 // incomplete declarations for the postcss-values-parser
 declare module 'postcss-values-parser' {
```

Comment:
I see that v5 comes with it's own types now - are they still incomplete? Can we open a PR back to upstream with these changes here?